packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Removes dependencies from `pubspec.yaml` that are only needed in `example/pubspec.yaml`
 * Updates Android compileSdkVersion to 31.
 * Internal code cleanup for stricter analysis options.
+* Fixes iOS crash on `EXC_BAD_ACCESS KERN_PROTECTION_FAILURE` if the map frame changes long after creation.
```

Comment:
You can't add changes to a version that has already been published. It needs a new version.